### PR TITLE
Composition: Fix list of versions missing current version

### PR DESCRIPTION
**Pull request by ndelangen**
## What I did
In the edge case `versions` is provided, but is missing the entry for the -current- selected version, storybook was crashing.

This prevents the crash, and just makes an entry called 'version'.

### CODE REVIEW

**Review by tmeasday — COMMENTED** [review]
Seems reasonable. Is there a story for this situation so we can see what it looks like / test it works?

**Comment by ndelangen** [comment]
I'll add a story

File: lib/ui/src/components/sidebar/RefIndicator.tsx [hunk 1]
```diff
@@ -102,10 +102,10 @@ const Version = styled.div({
 });
 
 const CurrentVersion: FunctionComponent<CurrentVersionProps> = ({ url, versions }) => {
-  const currentVersionId = useMemo(() => Object.entries(versions).find(([k, v]) => v === url)[0], [
-    url,
-    versions,
-  ]);
+  const currentVersionId = useMemo(() => {
+    const c = Object.entries(versions).find(([k, v]) => v === url);
+    return c && c[0] ? c[0] : 'version';
```
Comment (shilman) [inline lib/ui/src/components/sidebar/RefIndicator.tsx:107]:
```suggestion
    return c && c[0] ? c[0] : 'current';
```

**Review by shilman — APPROVED** [review]
Fix typo (I think)

**Pull request merged**